nextflow_schema.json

```
@@ -28,6 +28,12 @@
                     "description": "The output directory where the results will be saved. You have to use absolute paths to storage on Cloud infrastructure.",
                     "fa_icon": "fas fa-folder-open"
                 },
+                "vector_db": {
+                    "type": "string",
+                    "default": "/Users/gq2/git3/variantcalling/assets/vectorDB.tar.gz",
```

The default value here shouldn't be a path on your laptop :)
```suggestion
                    "default": "${projectDir}/assets/vectorDB.tar.gz",
```